r1.sh

```diff
@@ -65,20 +75,38 @@ python3 -m venv venv
 source venv/bin/activate
 
 chmod +x mtkbootcmd.py
+# Make `mtkbootcmd.py` use "${ESCAPED_PWD}"/venv/bin/python.
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    # Sed is different on mac.
+    grep -rl "usr/bin/env python3" mtkbootcmd.py | xargs sed -i '' $SEDSTR_ENVPYTHON
+elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    grep -rl "usr/bin/env python" mtkbootcmd.py | xargs sed -i $SEDSTR_ENVPYTHON
```

Comment:
mtkbootcmd.py shebang is already "usr/bin/env", active venv should take care of it

Reply:
I’ll check what happens on a fresh install of 20.04 or 22.04, to see if we can do this without manually patching all shebangs, seems hacky :)

Reply:
That sounds good.

That said, another way to see it is that replacing shebangs makes this repo resilient to changes mtkclient makes, and more hermetic since all references directly point to the virtualenv that is created by this code :)